src/Paket.Core/Installation/RestoreProcess.fs

```diff
@@ -438,10 +437,12 @@ let Restore(dependenciesFileName,projectFile,force,group,referencesFileNames,ign
                 s.Split([|';'|], StringSplitOptions.RemoveEmptyEntries)
                 |> Array.map (fun s -> s.Trim())
                 |> Array.choose FrameworkDetection.Extract)
+        
+        
+        let dependenciesFile = DependenciesFile.ReadFromFile(dependenciesFileName)
```

Comment:
So do we actually need a depsfile when this fails? Or should we warn and continue?